src/lavatory/commands/policies.py

```diff
@@ -32,12 +32,8 @@
 def policies(ctx, policies_path, repo, repo_type):
     """Prints out a JSON list of all repos and policy descriptions."""
     LOG.debug('Passed args: %s, %s, %s, %s', ctx, policies_path, repo, repo_type)
-    artifactory = Artifactory(repo_name=None)
-    all_repos = artifactory.list(repo_type=repo_type)
-    if repo:
-        selected_repos = repo
-    else:
-        selected_repos = all_repos.keys()
+
+    storage_info, selected_repos = get_artifactory_info(repo_name=None, repo_type=repo_type)
```

Comment:
If this only ever passes `repo_name = None`, how does it get repos specified in the CLI?